README.md

```diff
@@ -276,7 +276,9 @@ msync update -m "Commit message"
 Available parameters for modulesync.yml
 
 * `git_base` : The default URL to git clone from (Default: 'git@github.com:')
-* `namespace` : Namespace of the projects to manage (Default: 'puppetlabs')
+* `namespace` : Namespace of the projects to manage (Default: 'puppetlabs').
```

Comment:
Not blocking for this PR, but I think we should drop the default. Puppetlabs is no longer using modulesync so exactly 0 users benefit from this default.